docs/codeql/codeql-overview/codeql-changelog/codeql-cli-2.20.0.rst

```diff
@@ -0,0 +1,196 @@
+.. _codeql-cli-2.20.0:
+
+==========================
+CodeQL 2.20.0 (2024-12-09)
+==========================
+
+.. contents:: Contents
+   :depth: 2
+   :local:
+   :backlinks: none
+
+This is an overview of changes in the CodeQL CLI and relevant CodeQL query and library packs. For additional updates on changes to the CodeQL code scanning experience, check out the `code scanning section on the GitHub blog <https://github.blog/tag/code-scanning/>`__, `relevant GitHub Changelog updates <https://github.blog/changelog/label/code-scanning/>`__, `changes in the CodeQL extension for Visual Studio Code <https://marketplace.visualstudio.com/items/GitHub.vscode-codeql/changelog>`__, and the `CodeQL Action changelog <https://github.com/github/codeql-action/blob/main/CHANGELOG.md>`__.
+
+Security Coverage
+-----------------
+
+CodeQL 2.20.0 runs a total of 432 security queries when configured with the Default suite (covering 164 CWE). The Extended suite enables an additional 128 queries (covering 34 more CWE).
+
+CodeQL CLI
+----------
+
+New Features
+~~~~~~~~~~~~
+
+*   The |link-code-QlBuiltins-BigInt-type-1|_ of arbitrary precision integers is generally available and no longer hidden behind the
+    :code:`--allow-experimental=bigint` CLI feature flag.
+
+Known Issues
+~~~~~~~~~~~~
+
+*   The Windows executable for this release is labeled with an incorrect version number within its properties: the version number should be 2.20.0 rather than 2.19.4.
+    :code:`codeql version` reports the correct version number.
+
+Miscellaneous
+~~~~~~~~~~~~~
+
+*   Backslashes are now escaped when writing output in the Graphviz DOT format (:code:`--format=dot`).
+*   The build of Eclipse Temurin OpenJDK that is used to run the CodeQL CLI has been updated to version 21.0.5.
+
+Query Packs
+-----------
+
+Minor Analysis Improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+C/C++
+"""""
+
+*   The "Call to function with fewer arguments than declared parameters" query (:code:`cpp/too-few-arguments`) query no longer produces results if the function has been implicitly declared.
```

Comment:
```suggestion
*   The "Call to function with fewer arguments than declared parameters" query (:code:`cpp/too-few-arguments`) no longer produces results if the function has been implicitly declared.
```

Reply:
Do you know where the changenote originated? I'd like to modify the original too to avoid this showing up next time we generate the unified changelog.

Reply:
I think so, next time we build this we'll pick those updated changenotes to generate this unified one. Not sure my approval works for that, but happy to approve it.

Reply:
Your approval is probably sufficient.